src/index.html

```diff
@@ -9,10 +9,59 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
```

Comment:
Ensure that the linked stylesheet is the compiled CSS file from your SCSS source. The task requires SCSS to be compiled into CSS before being linked in the HTML.